packages/core/src/components/ui/ProductGallery/useDelayedFacets.ts

```diff
@@ -10,7 +10,7 @@ export const useDelayedFacets = (data?: ClientProductGalleryQueryQuery) => {
 
   return useMemo(() => {
     if (data) {
-      facets.current = data.search.facets
+      facets.current = data?.search?.facets
```

Comment:
`data?.search` is not needed because we've already checked if data is true-ish, right?

Reply:
nice catch! changed [here](https://github.com/vtex/faststore/pull/1967/commits/38d1479314ac0b7d14dbd607bef729b37f56e0f2).